doc/releases/changelog-0.34.0.md

```diff
@@ -288,19 +290,8 @@
 
```

Comment:
> Can't put this on the right line, but should we make more of the fact that you can return classical / non-measurements from circuits with Catalyst?

I'm on the fence, I think this would be nice to highlight once it works across all of PennyLane (not just Catalyst).